articles/architecture-scenarios/b2c/authentication.md

```diff
@@ -0,0 +1,120 @@
+* ...prevent hackers from trying to log in as my users?
+* ...implement authentication in different kinds of applications using Auth0?
+* ...make login easy for my users if they come from different language backgrounds?
+* ...provide a good user experience as I migrate away from my legacy authentication system?
+
+The answers to these questions, and others, are detailed below in sections on [Universal Login](#universal-login), [Username Password Authentication](#username-password-authentication), [Anomaly Detection](#anomaly-detection), and [Application Integration](#application-integration).
+
+## Universal Login
+
+Auth0's Universal Login makes authenticating users a short, three-step process (all of our [quickstarts](/quickstart) demonstrate this):
+
+1. Determine how and when you want to [redirect from your application](#application-integration).
+2. [Set up the appropriate branding and/or customized HTML](/universal-login) in your Auth0 configuration.
+3. Set up your application to [receive and handle the response](#application-integration) from the Authorization Server.
+
+::: note
+If you have more than one application, the best practice is to redirect to a centralized authorization server to authenticate the user (at Auth0, this means taking advantage of Universal Login). This comes with many security and user experience benefits, including SSO. See the [Universal Login](/universal-login) documentation for more information.
+:::
+
+## Username Password Authentication
+
+Nearly every B2C application provides that ability for their customers to create a new set of credentials for this company. This is a common form of authentication that all users are familiar with.
+
+Username Password Authentication can come in multiple flavors at Auth0. If your application is a greenfield application with no existing user base, then a simple Auth0 OOTB database connection will give you everything you need to start authenticating your users. However if you have a legacy user store (such as your own database of users or an existing LDAP system) you have a couple of different options for migrating your users, see here for more information.
+
+However you end up provisioning the users for your database connection, the authentication of those users is quite similar. It requires you to present the users with a form to enter their username and password. As mentioned in the universal login section, the simplest and safest way to authenticate users with a username and password is to redirect them to a centralized login page and collect their username and password there. This allows your authorization server to determine whether they have already authenticated and skip the login form entirely when it is not needed. See here for more information.
+
+::: note
+Collecting credentials only at the centralized login page will reduce the surface area for a potential leak of user secrets. It will also reduce the need to collect credentials unnecessarily.
```

Comment:
```suggestion
Collecting credentials only at the centralized login page will reduce the surface area for a potential leak of user secrets. It will also reduce the need to collect credentials unnecessarily.  See [Universal Login](#universal-login) for more information.
```